Integrate monitoring into piman.py

EveretteLi opened the pull request:
> enable "sudo python3 build/piman.pyz monitoring"
> put monitoring.config, host.csv into build folder in order to run monitoring from piman.pyz
> change the name of monitoring-client.py into monitoring_client to support the import in piman.py
> change piman.py. Start thread for monitoring on command "monitoring"
> check piman_README.md for more details

EveretteLi committed:
> enable "sudo python3 build/piman.pyz monitoring"
> put monitoring.config, host.csv into build folder in order ro run monitoring from piman.pyz
> change the name of monitoring-client.py into monitoring_client to support the import in piman.py
> change piman.py. Start thread for monitoring on command "monitoring"
> check piman_README.md for more details

breed commented on make_zipapp.sh:
```diff
@@ -8,6 +8,8 @@ mkdir build/piman.app
 )
 cp -r logging.conf utility tftp dhcp monitoring tcp *.py build/piman.app
+# copy hosts.csv and monitoring.config into build
+cp -r hosts.csv monitoring/monitoring.config build
```
hosts.csv should not go into build

breed commented on piman_README.md:
````diff
@@ -128,6 +128,23 @@ The official implementation is specified as `mapper(switch_address,interface, po
+``` python
+def monitoring():
+    monitor_thread= Thread(target = monitoring_client.start_from_piman, name= "monitoring")
+    monitor_thread.start()
````
you don't need to do the thread if you are just going to start and join

breed commented on piman.py:
```diff
@@ -106,6 +108,11 @@ def server():
 
+# start monitoring thread
+def monitoring():
+    monitor_thread= Thread(target = monitoring_client.start_from_piman, name= "monitoring")
```
you don't need a thread if you are just going to start and join

breed commented on monitoring/monitoring_client.py:
```diff
@@ -0,0 +1,115 @@
+"""
```
i think this is supposed to be a rename. note that there are changes to this file.

EveretteLi committed:
Update make_zipapp.sh

host.csv will no longer go to build

EveretteLi committed:
Updated piman_README.md

monitoring() no longer has a thread and just called start_from_piman from monitoring_client

EveretteLi committed:
Update monitoring_client.py

Added comments about the new method

EveretteLi committed:
Update piman.py

Deleted thread

EveretteLi commented:
The related changes have been made.

Thanks!
